lib/postgresql-store.js

```diff
@@ -15,6 +12,7 @@ var MAX_WAIT = 5000
 
 module.exports = function (opts) {
   var seneca = this
+  seneca.use(require('seneca-standard-query'))
```

Comment:
I thought a minute if this works here and i think it is. Seneca will require first `seneca-postgres` and then the standard query (`seneca-standard-query`) and only after both of them are initialised the ready event is triggered, I guess you can place it anywhere before the return :stuck_out_tongue: